osu.Game/Rulesets/Edit/EditorTimestampParser.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.RegularExpressions;
+using osu.Game.Rulesets.Objects;
+
+namespace osu.Game.Rulesets.Edit
+{
+    public static class EditorTimestampParser
+    {
+        // 00:00:000 (1,2,3) - test
+        // regex from https://github.com/ppy/osu-web/blob/651a9bac2b60d031edd7e33b8073a469bf11edaa/resources/assets/coffee/_classes/beatmap-discussion-helper.coffee#L10
+        public static readonly Regex TIME_REGEX = new Regex(@"\b(((\d{2,}):([0-5]\d)[:.](\d{3}))(\s\((?:\d+[,|])*\d+\))?)");
```

Comment:
This regex is still doing things like looking for digits, commas, or pipes. All of that should be down to the ruleset.

Reply:
Do we change that part to `[^)]` so it accepts _anything_ between the parentheses?
- `\b(((\d{2,}):([0-5]\d)[:.](\d{3}))(\s\((?:\d+[,|])*\d+\))?)`
- `\b(((\d{2,}):([0-5]\d)[:.](\d{3}))(\s\([^)]+\))?)`

Naturally, this includes the alphabet and other characters as well. It should be fine in terms of compatibility with Stable as it responds with an "Invalid format!" notification whenever you try to click on an unrecognized editor link.

If this is okay, we might also need to change this same regex in `osu-web` at:
https://github.com/ppy/osu-web/blob/3b1698639244cfdaf0b41c68bfd651ea729ec2e3/resources/js/utils/beatmapset-discussion-helper.ts#L78